Makefile

```diff
@@ -0,0 +1,5 @@
+gen-expected:
+	helm3 template --namespace=default --values=tests/version.yaml test . > tests/expected.yaml
```

Comment:
same here, would be better to have` HELM != helm3`

Reply:
+1 :-)